parent-pom/pom.xml

```diff
@@ -198,6 +198,12 @@
                     </exclusion>
                 </exclusions>
             </dependency>
+            <dependency>
+                <groupId>org.apache.spark</groupId>
+                <artifactId>spark-hadoop-cloud_${scala.compat.version}</artifactId>
+                <version>${spark.version}</version>
+                <scope>provided</scope>
+            </dependency>
```

Comment:
Since this is only specific to S3 variant, you could add this to `exasol-s3/pom.xml` file only.

Reply:
That's a good point, but after moving this dependency to s3, it started to complain about the security: https://github.com/exasol/spark-connector/actions/runs/5879323539/job/15943124384#step:7:3159

Not sure why it was silent before, is it checks for issues only dependency of connector-jdbc and connector-s3, but not parent's dependencies?

Reply:
Never mind, found the reason of those errors